attinyconfig.go

```diff
@@ -42,7 +42,7 @@ func ParseConfig(configDir string) (*AttinyConfig, error) {
 	location := config.DefaultWindowLocation()
 	rawConfig.Unmarshal(config.LocationKey, &location)
 
-	var battery config.Battery
+	battery := config.DefaultBattery()
```

Comment:
This can only work if go-config is updated right? I don't see the go-config update in this PR.